projects/go-style-guide/src/styles/_typography.scss

```diff
@@ -32,6 +32,9 @@ caption {
 
 .go-page-title {
   padding: 0 1rem 1rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
@Ashygoyal we try to keep our style attributes alphabetized for cleanliness, would you mind doing that here?

Reply:
Noted, and fixed.